public/style.css

```diff
@@ -35,3 +35,15 @@ textarea:focus {
 input {
   display: none;
 }
+
+@media (prefers-color-scheme: dark) {
+  body{
+    background-color:#202020;
+    color:#eee  ;
+  }
+
+  textarea{
+    background-color:#202020;
+    color:#eee  ;
+  }
```

Comment:
Here since both selectors have the same properties (same `background-color` and `color`) we can shorten the code a bit by applying the properties to both selectors at the same time:

```
body, textarea {
  …
}
```

We should be able to do the same with `public/html.css`.